From aa888fb289652f11d2bd86dafc4112b187249304 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Fri, 10 Oct 2025 22:06:21 +0300
Subject: [PATCH] fingerprint: Disable headers

---
 fingerprint/Android.bp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/fingerprint/Android.bp b/fingerprint/Android.bp
index 64d3a15..93a7e0a 100644
--- a/fingerprint/Android.bp
+++ b/fingerprint/Android.bp
@@ -10,7 +10,4 @@ cc_library_static {
     include_dirs: [
         "frameworks/native/services/surfaceflinger/CompositionEngine/include",
     ],
-    header_libs: [
-        "generated_kernel_headers",
-    ],
 }
-- 
2.53.0

