From ef6ada8b4d203a4bab1e351995d0432117fc1397 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Wed, 22 Oct 2025 21:46:56 +0300
Subject: [PATCH 2/4] build-Prefer-Etar-over-Calendar

---
 target/product/gsi/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/product/gsi/Android.bp b/target/product/gsi/Android.bp
index 2f67fb6..10106d4 100644
--- a/target/product/gsi/Android.bp
+++ b/target/product/gsi/Android.bp
@@ -179,7 +179,7 @@ android_system_image {
 
         // handheld_product
         "AvatarPicker",
-        "Calendar",
+        "Etar",
         "Contacts",
         "DeskClock",
         "Gallery2",
-- 
2.53.0

