From 895915b8b80eccd035b0d5119d585b39f8e404e8 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Sat, 18 Apr 2026 13:20:20 +0300
Subject: [PATCH 07/10] base: Remove non-existent healthd and vr_hwc modules

---
 base.mk | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/base.mk b/base.mk
index 2de7b50..5071165 100644
--- a/base.mk
+++ b/base.mk
@@ -196,9 +196,7 @@ PRODUCT_PACKAGES += \
     android.hardware.biometrics.fingerprint@2.1-service.oplus.compat \
 
 PRODUCT_PACKAGES += \
-    vr_hwc \
-    curl \
-    healthd \
+    curl
 
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     debug.fdsan=warn_once \
-- 
2.53.0

