From 02ae2a8b6ae488e16eac7387fca9e0efccfebe4e Mon Sep 17 00:00:00 2001
From: harvey186 <harvey186@hotmail.com>
Date: Thu, 7 May 2026 06:53:25 +0200
Subject: [PATCH 10/10] bugfix_PIN_setup_Pixel_devices

---
 phh-on-boot.sh | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/phh-on-boot.sh b/phh-on-boot.sh
index f05013f..1c7084c 100644
--- a/phh-on-boot.sh
+++ b/phh-on-boot.sh
@@ -101,15 +101,16 @@ if grep -qF android.hardware.boot /vendor/manifest.xml || grep -qF android.hardw
 	bootctl mark-boot-successful
 fi
 
-setprop ctl.restart sec-light-hal-2-0
-if find /sys/firmware -name support_fod |grep -qE .;then
-	setprop ctl.restart vendor.fps_hal
+### exclude Pixel devices
+if ! getprop ro.boot.hardware.platform | grep -qE -e '^gs(101|201|301)$' -e '^zuma' -e '^laguna'; then
+    setprop ctl.restart sec-light-hal-2-0
+    if find /sys/firmware -name support_fod |grep -qE .;then
+      setprop ctl.restart vendor.fps_hal
+    fi
+    setprop ctl.stop storageproxyd
+    sleep 10
 fi
 
-setprop ctl.stop storageproxyd
-
-sleep 10
-
 minijailSrc=/system/system_ext/apex/com.android.vndk.v28/lib/libminijail.so
 minijailSrc64=/system/system_ext/apex/com.android.vndk.v28/lib64/libminijail.so
 if [ "$vndk" = 27 ];then
-- 
2.53.0

